internal/luks2/cryptsetup_test.go

```diff
@@ -1009,3 +1015,41 @@ func (s *cryptsetupSuite) TestSetSlotPriority3(c *C) {
 		slotId:   1,
 		priority: SlotPriorityIgnore})
 }
+
+func (s *cryptsetupSuite) TestSelectCipherAndKeysize(c *C) {
```

Comment:
shouldn't we run this only one of the suites or a different suite?

Reply:
Thanks, I updated this and also improved/fixed the TestFormatOptionsValidateBadKeyslotsAreaSize test